app/src/main/AndroidManifest.xml

```diff
@@ -4,8 +4,6 @@
     package="com.mapbox.maps.testapp">
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
If we're talking about Android 30 - are we OK with just this permission? afair there were some changes but perhaps it's more about foreground / background location services.
But worth to read more I believe..

Reply:
yes, it definitely worth more investigation, and that's why I converted this PR to draft 😄